review-tools/pick-to-branch

```diff
@@ -11,7 +11,14 @@ function usage {
     A release branch may be given simply as 102, 110, 111, 30, 31."
 }
 
+num=0
+
 case $# in
+3)
+    id=$1
+    b=$2
```

Comment:
This overlaps with the $# == 2 case -
I suggest factoring out these joint assignments to avoid redundancy.

Reply:
I do not think that would really improve the readability of the code.